docs/getting-started/sdk/secrets-manager/integrations/kubernetes.mdx

```diff
@@ -0,0 +1,301 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Kubernetes
+
+The Bitwarden Secrets Manager Kubernetes Operator (sm-operator) is a tool for teams to integrate
+Bitwarden Secrets Manager into their Kubernetes workflows seamlessly.
+
+The sm-operator uses a
+[controller](https://github.com/bitwarden/sm-kubernetes/blob/main/src/sm-operator/internal/controller/bitwardensecret_controller.go)
+to synchronize Bitwarden Secrets into Kubernetes secrets. It does so by registering a Custom
+Resource Definition of BitwardenSecret into the cluster. It will listen for new BitwardenSecrets
+registered on the cluster and then synchronize on a configurable interval.
+
+:::info
+
+If you're new to Secrets Manager, you should read the [Help Center documentation][help] first to
```

Comment:
⛏️ This is the only place you use a reference vs. an inline link.

Reply:
I pulled this basically from the key connector article.  I was trying to follow that route.

Reply:
Updated.